mytimer/functions.py

```diff
@@ -7,10 +7,24 @@
 from mytimer.params import INPUT_ERROR_MESSAGE, SOUND_ERROR_MESSAGE
 from mytimer.params import INPUT_EXAMPLE, TIME_ELEMENTS, MESSAGE_TEMPLATE
 from mytimer.params import FACES_MAP, PROGRAMS_MAP, TONES_MAP
-from mytimer.params import MY_TIMER_VERSION
+from mytimer.params import MY_TIMER_VERSION, PROGRAMS_LIST_TEMPLATE
 from art import tprint
 
 
+def _show_programs_list():
+    """
+    Show programs list.
+
+    :return: None
+    """
```

Comment:
To improve the clarity of the command, it would be beneficial to include a title that clearly identifies the subject matter or purpose.
```python
print("Programs list:\n\n")
```